core/package.yaml

```diff
@@ -0,0 +1,65 @@
+name:       rio-core
+synopsis:   Minimal version of rio with the RIO core type.
+homepage:   https://github.com/strainge/rio-core#readme
+maintainer: Allan Lukwago <epicallan.al@gmail.com>, Markus Schirp mbj@schirp-ds.com
+copyright:  2020 Strainge Inc.
+license:    MIT
+github:     strainge/rio-core
+version:    0.0.1
+
+dependencies:
+- base >= 4.10 && < 10
```

Comment:
Declaring we are compatible with any base upto excluding `10` seems to be very wide, is this a good default?

Reply:
5 should be a good compromise. This was done in an attempt to match `rio's` base constraints.

Reply:
@epicallan I'll delegate to your experience given RIO uses this also.